src/awkward/_v2/cpp-headers/awkward/GrowableBuffer.h

```diff
@@ -304,9 +304,9 @@ namespace awkward {
     void
     append(PRIMITIVE datum) {
       if (ptr_->length() == ptr_->reserved()) {
-        add_panel((size_t)ceil(ptr_->reserved() * 1.5));
+        _add_panel((size_t)ceil(ptr_->reserved() * 1.5));
```

Comment:
@ManasviGoyal - Certain sets of names and function signatures are always reserved to the implementation in C++. I’d avoid using leading underscore for both function names and private members. When in doubt, please, use Google C++ style guide: https://google.github.io/styleguide/cppguide.html

Reply:
@ianna Hi. I think you check one of the previous commits. I have removed the underscores already.